modules/lmdbbackend/Makefile.am

```diff
@@ -4,6 +4,7 @@ pkglib_LTLIBRARIES = liblmdbbackend.la
 
 EXTRA_DIST = OBJECTFILES OBJECTLIBS
 
-liblmdbbackend_la_SOURCES = lmdbbackend.cc lmdbbackend.hh lmdb-safe.hh lmdb-safe.cc lmdb-typed.hh lmdb-typed.cc
+liblmdbbackend_la_SOURCES = lmdbbackend.cc lmdbbackend.hh lmdb-typed.hh lmdb-typed.cc \
+	../../pdns/lmdb-safe.hh ../../pdns/lmdb-safe.cc
```

Comment:
so if lmdb-safe.* come from ahu's repo, and this should continue, should the files live in ext/lmdb-safe/ instead?

Reply:
That would make sense, yes. I guess we that applies to lmdb-typed.{cc,hh} as well.

Reply:
Done!